### PR TITLE
V2V Support for SN24

### PR DESCRIPTION
**Pull request by square-1111**
Todos:
- [ ]: Evaluation scheme for youtube audios
    - [x] : Prototyping
- [ ]: Designing the system for the same

### CODE REVIEW

File: neurons/miner_v2v.py
Comment (xzistance) [inline neurons/miner_v2v.py:1]: @square-1111 is this file needed any longer now that you combined them to `neurons/miner.py` ?

File: omega/base/validator.py
Comment (xzistance) [inline omega/base/validator.py:1]: @square-1111 - you're going to need to put a check down at the bottom of this file in the `load_state` function similar to how we did focus_scores so that validators don't crash when they load the update for the first time and the audio scores variable is not in their state file.

File: omega/diarization_pipeline.py
Comment (xzistance) [inline omega/diarization_pipeline.py:1]: @square-1111 - is this file only used by miners?

File: omega/diarization_metric.py
Comment (xzistance) [inline omega/diarization_metric.py:1]: @square-1111 - is this file only used by miners?

File: neurons/miner_v2v.py
Comment (square-1111) [reply neurons/miner_v2v.py:1]: yeah this can be removed. on it

File: omega/diarization_pipeline.py
Comment (square-1111) [reply omega/diarization_pipeline.py:1]: Nope. it will also be used by validators to compute score

File: omega/diarization_metric.py
Comment (square-1111) [reply omega/diarization_metric.py:1]: this is only used by validators, not by miners

**Comment by square-1111** [comment]
@xzistance take a look. Made few more changes

File: omega/diarization_pipeline.py
Comment (xzistance) [reply omega/diarization_pipeline.py:1]: OK, the reason I ask is because of the HF TOKEN stuff. Is an HF Token necessary for this to work? What happens if a validator doesn't have one?

Do these repos require a token?
`diarization_model_id = "Revai/reverb-diarization-v2"
overlap_detection_model_id = "pyannote/overlapped-speech-detection" `

Comment (square-1111) [reply omega/diarization_pipeline.py:1]: Yeah, `diarization_model_id = "Revai/reverb-diarization-v2"` uses HF_TOKEN.
